mountinfo/mountinfo_filters.go

```diff
@@ -6,9 +6,8 @@ import "strings"
 // used to filter out mountinfo entries we're not interested in,
 // and/or stop further processing if we found what we wanted.
 //
-// It takes a pointer to the Info struct (not fully populated,
-// currently only Mountpoint, FSType, Source, and (on Linux)
-// VFSOptions are filled in), and returns two booleans:
+// It takes a pointer to the Info struct (fully populated with all available
+// fields on the GOOS platform), and returns two booleans:
```

Comment:
Wondering ; is the "on the GOOS platform" part relevant in the comment? Otherwise we could simplify to;

```suggestion
// It takes a pointer to the populated Info struct, and returns two booleans:
```

Reply:
I guess the point I was trying to make is that on FreeBSD, not all of the fields are actually filled because some of the `Info` struct fields are Linux-only concepts.

Reply:
Ah, right, and they use the same struct 🤔 gotcha!

It was mostly a suggestion, trying to keep it simple, definitely not a blocker 👍